stack-8.8.3.yaml

```diff
@@ -5,66 +5,6 @@ extra-deps:
 - equivalence-0.3.5
 - geniplate-mirror-0.7.6
 - STMonadTrans-0.4.4
-- attoparsec-0.13.2.3
-- base-compat-0.11.1
-- blaze-builder-0.4.1.0
-- blaze-markup-0.8.2.3
-- dlist-0.8.0.7
-- fail-4.9.0.0
-- network-uri-2.6.3.0
-- primitive-0.7.0.0
-- random-1.1
-- regex-base-0.94.0.0
-- scientific-0.3.6.2
-- tagged-0.8.6
-- th-abstraction-0.3.2.0
-- time-compat-1.9.2.2
-- transformers-compat-0.6.5
-- utf8-string-1.0.1.1
-- uuid-types-1.0.3
-- vector-0.12.1.2
-- base-orphans-0.8.2
-- integer-logarithms-1.0.3
-- QuickCheck-2.13.2
-- filemanip-0.3.6.3
-- process-extras-0.7.4
-- tasty-1.2.3
-- tasty-hunit-0.10.0.2
-- tasty-quickcheck-0.10.1.1
-- tasty-silver-3.1.13
-- temporary-1.3
-- unix-compat-0.5.2
-- ListLike-4.6.3
-- ansi-terminal-0.10.3
-- call-stack-0.2.0
-- clock-0.8
-- data-default-0.7.1.1
-- generic-deriving-1.13.1
-- optparse-applicative-0.15.1.0
-- semigroups-0.19.1
-- splitmix-0.0.4
-- unbounded-delays-0.1.1.0
-- wcwidth-0.0.2
-- ansi-wl-pprint-0.6.9
-- colour-2.3.5
-- data-default-class-0.1.2.0
-- data-default-instances-containers-0.0.1
-- data-default-instances-dlist-0.0.1
-- data-default-instances-old-locale-0.0.1
-- fmlist-0.9.3
-- old-locale-1.0.0.7
-- Diff-0.4.0
-- HUnit-1.6.0.0
-- cmdargs-0.10.20
-- pretty-show-1.10
-- safe-0.3.18
-- test-framework-0.8.2.0
-- test-framework-hunit-0.3.0.2
-- haskell-lexer-1.1
-- hostname-1.0
-- regex-posix-0.96.0.0
-- xml-1.3.14
-- extensible-exceptions-0.1.1.4
 - bifunctors-5.5.7
 - comonad-5.0.6
 - cabal-doctest-1.0.8
```

Comment:
The libraries `bifunctors-5.5.7`,  `comonad-5.0.6`, and `cabal-doctest-1.0.8` are included in [lts-15.4](https://www.stackage.org/lts-15.4). Please remove them.